qemu/tests/cfg/pktgen.cfg

```diff
@@ -19,11 +19,12 @@
             image_snapshot = yes
         - host_guest:
             #host as server
-            password_pktgen_server = redhat
+            # please modify this password according to your setting as below
+            # password_pktgen_server = <your password>
             shell_prompt_pktgen_server =  \[root@.{0,50}][\#\$]
         - exthost_guest:
             #exthost as server
-            pktgen_server = 10.66.9.48
+            # pktgen_server = 10.66.9.48
             server_interface = switch
-            password_pktgen_server = redhat
+            # password_pktgen_server = redhat
```

Comment:
```python
# pktgen_server = <your server ip>
# password_pktgen_server = <your password>
```

Reply:
Updated. Please help to check again. Thanks.